server/src/models/equipment/equipment.ts

```diff
@@ -1,12 +1,7 @@
-import { EquipmentLabel } from "./equipmentLabel";
-
 export interface Equipment {
-    id: number;
-    name: string;
-    room: string;
-    addedAt: Date,
-    inUse: boolean;
+  id: number;
+  name: string;
+  room: string;
```

Comment:
Just make sure to fix this since room is now roomID and an int instead of string
```roomID: int
  room: string;
```

Reply:
Fixed with merging master into my branch 👍